### PR TITLE
Feature/m0078 ddg 342 move target folder

### PR DESCRIPTION
**Pull request by gjadi-mej**

### CODE REVIEW

File: Tutorials/tutorialSetupBuildUsingJenkinsAndArtifactory.rst [hunk 1]
```diff
@@ -447,4 +447,18 @@ The authority has to be added to the trust store of the JRE/JDK that is running
...
 #. Save the ``cacerts`` file.
 
+Customize ``target~`` path
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Some systems and toolchains don't handle long path properly.  A
+workaround for this issue is to move the build directory (that is, the
+``target~`` directory) closer to the root directory.
+
+To change the ``target~`` directory path, override the Ant property ``target``.
```
Comment (frivieremicroej) [inline Tutorials/tutorialSetupBuildUsingJenkinsAndArtifactory.rst:457]: set the <build_option> ``target``

**Comment by gjadi-mej** [comment]
- https://docs.microej.com/en/feature-m0078ddg-342_move_target_folder/ApplicationDeveloperGuide/mmm.html#build-options
- https://docs.microej.com/en/feature-m0078ddg-342_move_target_folder/Tutorials/tutorialSetupBuildUsingJenkinsAndArtifactory.html#customize-target-path

File: Tutorials/tutorialSetupBuildUsingJenkinsAndArtifactory.rst (hunk 1 above)
Comment (gjadi-mej) [reply Tutorials/tutorialSetupBuildUsingJenkinsAndArtifactory.rst]: Added

**Comment by gjadi-mej** [comment]
Closing.  Integrated in #149

**Pull request closed without merging**